src/spryker/harness/scripts/enable.sh

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+
+main()
+{
+    if [ ! -f .my127ws/.flag-built ]; then
+    
+        passthru docker-compose down
+
+        if [[ "$HAS_ASSETS" = "yes" ]]; then
+            ws assets download
+        fi
+
+        $APP_BUILD
+        touch .my127ws/.flag-built
+
+    else
+        passthru docker-compose up -d
+        passthru docker-compose exec -T -u build console app welcome
+    fi
+
+    if [[ "$APP_BUILD" = "dynamic" && "$USE_DOCKER_SYNC" = "yes" ]]; then
+        passthru docker-sync start
+    fi
+}
+
+dynamic()
+{
+    # we synchronise then stop docker-sync as leaving it running during the build
+    # will often cause it to crash.
+
+    if [[ "$USE_DOCKER_SYNC" = "yes" ]]; then
+        passthru docker-sync start
+        passthru docker-sync stop
+    fi
+
+    # this is causing issue with setting a local image
+    # passthru docker-compose pull
+    # passthru docker-compose build --pull
+    passthru docker-compose build
```

Comment:
I've tag console image as ":dev" in local and using it as base image for jenkins-slave, because of this it gives an error when it tries to pull the images first (as local image for console doesn't exist yet), so I had to comment pull command. Is it ok to skip pull as build would do it anyway if image doesn't exist? or how should we handle this?

Reply:
It would be better to still have the pull to avoid stale images causing inconsistent builds between people.

Perhaps for the first pull:
```bash
docker-compose config --services | grep -v console | xargs docker-compose pull
```

and for the build --pull:
```bash
docker-compose config --services | grep -v jenkins-runner | xargs docker-compose build --pull
docker-compose build jenkins-runner
```